docs/hpc/about/pricing-compute.md

```diff
@@ -1,33 +1,52 @@
 
-* CPU/node: 2 × Intel Xeon Gold 6336Y 2.4 GHz 24-core processors
-* RAM/node: 256 GiB
+**Price per 4 compute node unit: ~$33k USD excluding tax (for 4 nodes in a 2U chassis)**
 
-**Price per 4 compute node unit: 33,405 USD (for 4 nodes in a 2U chassis)**
+SMC RM224Q 2U Quad-Node Server 
+Four Hot-Pluggable Compute Modules, each Node Includes:  
+CPU: (2) Xeon Gold 6336Y 2.4GHz Twenty-Four-core  
+Memory: 256GB DDR4-3200 ECC (16x 16GB)  
+OS: (1) 1.92TB SATA SSD, 3DWPD 
 
 Based on the above pricing, the pricing for CPU contributions is about $173 per CPU/Slot/Share.
 
+**As of May 2023, lead times for compute nodes are 6 weeks from order to delivery.** Once the hardware is received, we will schedule time to configure hardware as time allows given priority projects. Unless you are purchasing a 4-compute node unit, we will need to wait until we have enough requests to place an order for a whole unit. If we have capacity, we may be able to provision some shares in advance of the billing. 
+
 ## Pricing for Graphics Processing Unit (GPU)
 
 <div class="alert alert-info" role="alert" markdown="1">
 {{ site.cluster.name }} has {{ site.data.specs.gpu_nodes }} GPU nodes with a total of {{ site.data.specs.gpus }} GPUs available to all users. Among these, {{ site.data.specs.gpu_nodes | minus: site.data.specs.communal_gpu_nodes }} GPU nodes, with a total of {{ site.data.specs.gpus | minus: site.data.specs.communal_gpus }} GPUs, were contributed by different research groups. GPU jobs are limited to 2 hours in length when run on GPUs not contributed by the running user's lab.  In contrast, [contributors are _not_ limited to 2-hour GPU jobs on nodes they contributed](/hpc/scheduler/queues.html). There is also one GPU development node that is available to all users.
 </div>
 
-* CPU: 1 × AMD EPYC 7543P 2.8 GHz 32-core processor
-* GPU: 4 × Nvidia A40
-* RAM: 512 GiB/node
+Single GPU cards cannot be purchased - only a full GPU node. However, just like partial compute nodes can be purchased, GPU purchase requests can also be pooled towards a full system. 
+
+**All options include the following Mercury GPU208 2U Server:** 
+CPU: (1) EPYC 7543P 2.8GHz Thirty-Two-Core Procs 
+Memory: 512GB DDR4-3200 (8x 64GB)  
+OS Drive: (1) 512GB SATA SSD  
+Storage: (1) 960GB U.2 NVMe SSD  
+
+**Price per A40 GPU node: ~$25k USD (including warranty, excluding partial tax)**
+GPU: (4) A40 48GB GDDR6
 
-**Price per GPU node:  25,412 USD**
+**Price per L40 GPU node: ~$33k USD (including warranty, excluding partial tax)**
+GPU: (4) L40 48GB GDDR6 PCIe
 
+**Price per A100 GPU node: ~$58k USD (including warranty, excluding partial tax)**
+GPU: (4) A100 80GB GDDR6 PCIe 
+
+**As of May 2023, lead times for GPUs are 4-5 months from order to delivery.** Once the hardware is received, we will schedule time to configure hardware as time allows given priority projects. Once the hardware is received, we will schedule time to configure hardware as time allows given priority projects.  
```

Comment:
The "Once the hardware is received, we will schedule time to configure hardware as time allows given priority projects." sentence is duplicated.

Reply:
oops - thanks for catching

Reply:
FYI, you can edit this on your end.  When you commit your edits, they'll automatically end up here in this pull request.